charts/tautulli/item.yaml

```
@@ -1,3 +1,3 @@
 categories:
   - media
-icon_url: https://pirotta.digital/images/tautulli-image.png
+icon_url: https://github.com/Tautulli/Tautulli/blob/master/data/interfaces/default/images/logo-circle.png
```

Maybe just use the logo without background?
```suggestion
icon_url: https://github.com/Tautulli/Tautulli/blob/master/data/interfaces/default/images/logo.png
```

---

Yes, better